.github/workflows/docker-ingestion.yml

```diff
@@ -99,13 +99,13 @@ jobs:
       - name: Download image
         uses: ishworkh/docker-image-artifact-download@v1
         with:
-          image: acryldata/datahub-ingestion-slim:latest
```

Comment:
This does not seem correct. Please see https://github.com/datahub-project/datahub/pull/7653 from @shirshanka